contracts/test/modules/CollectionOffers/V1/CollectionOffers.integration.t.sol

```diff
@@ -0,0 +1,168 @@
+
+        // Mint buyer token
+        token.mint(address(buyer), 0);
+
+        // Users approve Collection Offers module
+        seller.setApprovalForModule(address(offers), true);
+        buyer.setApprovalForModule(address(offers), true);
+
+        // Buyer approve ERC721TransferHelper
+        vm.prank(address(buyer));
+        token.setApprovalForAll(address(erc721TransferHelper), true);
+    }
+
+    /// ------------ ETH COLLECTION OFFER ------------ ///
+
+    function offer() public {
+        vm.prank(address(seller));
+        offers.createOffer{value: 1 ether}(address(token));
+    }
+
+    function fill() public {
+        vm.prank(address(seller));
+        offers.createOffer{value: 1 ether}(address(token));
+
+        vm.prank(address(buyer));
+        offers.fillOffer(address(token), 0, 1 ether, address(finder));
+    }
+
+    function test_WithdrawOfferFromSeller() public {
+        uint256 beforeBalance = address(seller).balance;
+        offer();
+        uint256 afterBalance = address(seller).balance;
+
+        require(beforeBalance - afterBalance == 1 ether);
+    }
+
+    function test_WithdrawOfferIncreaseFromSeller() public {
+        uint256 beforeBalance = address(seller).balance;
+
+        offer();
+
+        // Increase initial offer to 2 ETH
+        vm.prank(address(seller));
+        offers.setOfferAmount{value: 1 ether}(address(token), 1, 2 ether);
+
+        uint256 afterBalance = address(seller).balance;
+
+        require(beforeBalance - afterBalance == 2 ether);
+    }
+
+    function test_RefundOfferDecreaseToSeller() public {
+        uint256 beforeBalance = address(seller).balance;
+
+        offer();
+
+        // Decrease initial offer to 0.5 ETH
+        vm.prank(address(seller));
+        offers.setOfferAmount(address(token), 1, 0.5 ether);
+
+        uint256 afterBalance = address(seller).balance;
+
+        require(beforeBalance - afterBalance == 0.5 ether);
+    }
+
+    function test_ETHIntegration() public {
+        uint256 beforeSellerBalance = address(seller).balance;
+        uint256 beforeBuyerBalance = address(buyer).balance;
+        uint256 beforeRoyaltyRecipientBalance = address(royaltyRecipient).balance;
+        uint256 beforeFinderBalance = address(finder).balance;
+        address beforeTokenOwner = token.ownerOf(0);
+
+        fill();
+
+        uint256 afterBuyerBalance = address(buyer).balance;
+        uint256 afterSellerBalance = address(seller).balance;
+        uint256 afterRoyaltyRecipientBalance = address(royaltyRecipient).balance;
+        uint256 afterFinderBalance = address(finder).balance;
+        address afterTokenOwner = token.ownerOf(0);
+
+        // 1 ETH withdrawn from seller
+        require((beforeSellerBalance - afterSellerBalance) == 1 ether);
+        // 0.05 ETH creator royalty
+        require((afterRoyaltyRecipientBalance - beforeRoyaltyRecipientBalance) == 0.05 ether);
+        // 100 bps finders fee (Remaining 0.95 ETH * 10% finders fee = 0.0095 ETH)
```

Comment:
A default finders fee is configured in `CollectionOffersV1`'s constructor:

https://github.com/ourzora/v3/blob/78ce19a325b350bb18c5a844e73bfad588497214/contracts/modules/CollectionOffers/V1/CollectionOffersV1.sol#L91

Since the maximum finders fee is `10000`, a `100` bps finders fee is `1%`, not `10%`. 

It follows that `0.95 ETH` * `1%`  is `0.0095 ETH` as it's expressed in the assertion.

Given this context, the following annotation should be updated accordingly for correctness:

```suggestion
        // 100 bps finders fee (Remaining 0.95 ETH * 1% finders fee = 0.0095 ETH)
```